tests/test_project.py

```diff
@@ -1,18 +1,46 @@
+import pytest
 import subete
 
 
 def test_project_str():
     test = subete.Project("hello-world", {"words": ["hello", "world"], "requires_parameters": False})
     assert str(test) == "Hello World"
 
+
 def test_name():
     test = subete.Project("hello-world", {"words": ["hello", "world"], "requires_parameters": False})
     assert test.name() == "Hello World"
 
+
 def test_pathlike_name():
     test = subete.Project("hello-world", {"words": ["hello", "world"], "requires_parameters": False})
     assert test.pathlike_name() == "hello-world"
 
+
+@pytest.mark.parametrize("name", ["import-blah", "blah-export"])
+def test_pathlike_name_when_import_or_export(name):
+    test = subete.Project(name, {"words": name.split("-"), "requires_parameters": False})
+    assert test.pathlike_name() == "import-export"
```

Comment:
OK, this is weird. Why does the presence of "import" or "export" in the project name cause the project name to change to "import-export"?

Reply:
This is a nonissue now since we no longer support the import-export project. It was too much of a nightmare to maintain, and it wasn't possible across most of the languages. I think anything referencing this can be removed.